base-pom/pom.xml

```
@@ -36,9 +36,9 @@
   <properties>
     <xwiki.version>2.7.2</xwiki.version>
     <spring.version>5.3.27</spring.version>
```

```suggestion
    <spring.version>5.3.39</spring.version>
```
can we bump this to the latest 5.3 bugfix?